docs/get-started/using-with-map.md

```diff
@@ -29,7 +29,7 @@ All the [examples on this website](https://github.com/visgl/deck.gl/tree/master/
 
 ### Using Mapbox basemap service (with Mapbox token)
 
-The mapbox-gl library is open source and free to use. However, to load the map styles and tiles from Mapbox's data service, you will need to register on their website in order to retrieve an [access token](https://docs.mapbox.com/help/how-mapbox-works/access-tokens/) required by the map component, which will be used to identify you and start serving up map tiles. The service will be free until a certain level of traffic is exceeded.
+The mapbox-gl library is open source and free to use. However, to load the map styles and tiles from Mapbox's data service, you will need to register on their website in order to retrieve an [access token](https://docs.mapbox.com/help/how-mapbox-works/access-tokens/) required by the map component, which will be used to identify you and start serving up map tiles. The service will be free until a [certain level](https://docs.mapbox.com/api/#rate-limits) of traffic is exceeded.
```

Comment:
I don't think the link to `rate-limits` is the right one... that is irrelevant of what's free. Maybe link to pricing? https://www.mapbox.com/pricing/